android/app/src/main/java/updated/mysterium/vpn/model/manual/connect/ConnectionState.kt

```diff
@@ -5,5 +5,5 @@ enum class ConnectionState(val state: String) {
     CONNECTING("Connecting"),
     CONNECTED("Connected"),
     DISCONNECTING("Disconnecting"),
-    ON_HOLD("OnHold")
+    ONHOLD("OnHold")
```

Comment:
Like here you divide words with camel case

Reply:
Fix